client/my-sites/earn/memberships/index.jsx

```diff
@@ -543,7 +605,11 @@ class MembershipsSection extends Component {
 					shouldDisplay={ () => true }
 					feature={ FEATURE_MEMBERSHIPS }
 					title={ this.props.translate( 'Upgrade to the Personal plan' ) }
-					description={ this.props.translate( 'Upgrade to start earning recurring revenue.' ) }
+					description={
+						isEnabled( 'earn/rename-payment-blocks' )
+							? this.props.translate( 'Upgrade to start selling.' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Upgrade My Site + Start Selling' )` *ES Score: 6*